aikau/src/main/resources/alfresco/services/DocumentService.js

```diff
@@ -184,21 +184,10 @@ define(["dojo/_base/declare",
             var nodeRef = NodeUtils.processNodeRef(payload.nodeRef),
             targetNodeUri = nodeRef.uri;
 
-            // Construct the URI for the request...
```

Comment:
Thanks for highlighting this and removing it - I'd not spotted it was not being used.